src/operator/tensor/elemwise_unary_op_trig.cc

```diff
@@ -201,7 +202,35 @@ The storage type of ``arctan`` output depends upon the input storage type:
 .set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseIn{ "_backward_arctan" });
 
 MXNET_OPERATOR_REGISTER_BINARY_WITH_SPARSE_CPU_DR(_backward_arctan,
-                                                  unary_bwd<mshadow_op::arctan_grad>);
+                                                  unary_bwd<mshadow_op::arctan_grad>)
+.set_attr<nnvm::FGradient>("FGradient",
+    [](const nnvm::NodePtr& n, const std::vector<nnvm::NodeEntry>& ograds) {
+      // ograds[0]: head_grad_grads (dL/dy_grad)
```

Comment:
Isn't it dL/dx_grad?

Reply:
Please have a look and let me know what you think.
https://github.com/apache/incubator-mxnet/pull/15331#discussion_r304541519